pcsx2/GS/GSCrc.cpp

```diff
@@ -477,24 +482,50 @@ CRC::Game CRC::m_games[] =
 	{0xCE4933D0, RatchetAndClank, US, 0},
 	{0x6F191506, RatchetAndClank, US, 0}, // E3 Demo
 	{0x81CBFEA2, RatchetAndClank, US, 0}, // EB Games Demo
+	{0xAAAA62BE, RatchetAndClank, US, 0}, // 2306 Beta
+	{0xBE03FCFA, RatchetAndClank, US, 0}, // 2506 Beta
+	{0x660CDF9A, RatchetAndClank, US, 0}, // 0208 Beta
+	{0xA6FEE57C, RatchetAndClank, US, 0}, // 0909 Beta
 	{0x56A35F77, RatchetAndClank, JP, 0},
 	{0x76F724A3, RatchetAndClank, EU, 0},
 	{0x6A8F18B9, RatchetAndClank, EU, 0},
+	{0x5C19F3B7, RatchetAndClank, EU, 0}, // Regular Demo
 	{0xB3A71D10, RatchetAndClank2, US, 0}, // Going Commando
 	{0x38996035, RatchetAndClank2, US, 0},
+	{0xF700EE7E, RatchetAndClank2, US, 0}, // Regular Demo
+	{0xF67ADF58, RatchetAndClank2, US, 0}, // Retail Employees Demo
```

Comment:
remove

Reply:
You want me to remove just the comments and not the entire line right?

Reply:
I want you to remove the entire line 496 but 495 can stay. If I want you to remove multiple lines it'll say something like `Comment on lines x to y` right above the comment thread.

Reply:
Okay, but is there any specific reason why I have to remove entire lines for the beta and demo versions? For the beta versions I can understand why you wouldn't want to support them since they are non-retail versions, but since there were already CRCs for some of them (like the 1108 Beta of UYA) I assumed it would've been fine to add the rest of them. The demo and public beta versions however should absolutely stay since they were distributed in retail.

Reply:
I don't want protos for legal and maintenance reasons. If there's already CRCs for that's our mistake and they need to go. As for demos I don't mind them but I don't know what "Retail Employees Demo" is and I error on the side of safety. If you explain what it is, then I'll reconsider. I don't believe I asked you to remove any of the lines labeled "public beta".

Reply:
Alright, I'll remove the beta versions then. As for the Retail Employees Demo (SCUS-97323), I assume it was distributed in retail game stores as a playable demo which means it was publicly available. It is on the PCSX2 wiki too as well, so I think it should be fine to leave it in.